src/main/java/ro/hiringsystem/controller/PositionsController.java

```diff
@@ -0,0 +1,18 @@
+package ro.hiringsystem.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ro.hiringsystem.model.enums.Position;
+
+@RestController
+@RequestMapping("/api/v1/positions")
```

Comment:
Same comment here as for jobTypes, I suggest mapping them in the JobsController via "api/v1/job/positions" or "api/v1/job/positions/get"